src/libs/OptimisticReportNames.ts

```diff
@@ -362,6 +396,36 @@ function updateOptimisticReportNamesFromUpdates(updates: OnyxUpdate[], context:
                 break;
             }
 
+            case 'personalDetails': {
```

Comment:
We don't need to recompute when personal details are updated. The user level fields are supposed to be snapshots of the value at the time of submission.